src/types/object.rs

```diff
@@ -103,6 +103,17 @@ impl ZendObject {
         unsafe { self.ce.as_ref() }.expect("Could not retrieve class entry.")
     }
 
+    /// Returns the [`ClassEntry`] associated with this object.
+    ///
+    /// # Panics
+    ///
+    /// Panics if the class entry is invalid.
+    pub fn get_class_entry_mut(&self) -> &'static mut ClassEntry {
```

Comment:
?

Reply:
I need a `& mut class entry` to get the iterator, not sure if this is best way to retrieve it otherwise i can directly get it from the raw pointer of the object